fastavro/_read_py.py

```diff
@@ -478,34 +479,68 @@ def skip_sync(fo, sync_marker):
         raise ValueError('expected sync marker not found')
 
 
+class ReaderBase(object):
+    def __init__(self):
+        self._position = 0
+
+    def position(self):
```

Comment:
Should this be changed to `tell` to make it more similar to a normal `io` object?